tests/tests_pytorch/models/test_restore.py

```diff
@@ -311,9 +311,11 @@ def get_trainer_args():
                 "best_k_models",
                 "kth_best_model_path",
                 "kth_value",
-                "last_model_path",
             ):
-                assert getattr(before, attribute) == getattr(after, attribute)
+                assert getattr(before, attribute) == getattr(after, attribute), f"{attribute}"
+            # `before.last_model_path` is a symlink pointing to a checkpoint saved before that symlink was created,
+            # hence reloading that checkpoint will restore `after.last_model_path = ""`
+            assert after.last_model_path == ""
```

Comment:
Isn't this a problem?

Reply:
Depends, we can make it a problem and change it, or we can say this is expected. Depends on the definition of it. It's difficult to handle though. One way would be to format and set the path `last_model_path` before it is being created, so that it gets stored in the checkpoint ahead of its creation. Would we be comfortable with that and could it lead to unexpected edge cases? I'm not sure